### ISSUE TITLE
Accessibility metadata available for files

### ISSUE THREAD

**Issue by flittle8** [BCLibCoop/nnels#152]
To meet [EPUB Accessibility 1.0 ](http://www.idpf.org/epub/a11y/accessibility.html)requirements, we need to make available accessibility metadata for our files in Drupal. 
We should be able to extract this metadata from the file into Drupal, and also be able to manually add this metadata into fields in Drupal. The data should be exposed to users so they know what accessibility features the file contains and lacks.
Read more details about schema.org accessibility metadata on W3C wiki (https://www.w3.org/wiki/WebSchemas/Accessibility), [schema.org](http://schema.org), and on the [NNELS wiki](https://wiki.libraries.coop/doku.php?id=cooperative:users:farrah:metadata).

For **EPUB files**, the accessibility metadata would be inside the .opf file (it will either be provided by the publisher or added into the  .opf by NNELS).
For **DAISY files**, I think NNELS can add accessibility metadata into the .opf file (for DAISY 3) or the ncc.html file (for DAISY 2.02); however I'm not sure how to do this (suggestions?).
For **other file formats** (MP3, etext, braille, etc.), I think the metadata will need to be manually added into the new Drupal fields associated with the file resource. 

Note: In the future, accessibility metadata will also be made available in ONIX and MARC files provided by the publisher or library. See [crosswalk](http://www.a11ymetadata.org/the-specification/metadata-crosswalk/). [ONIX](https://onix-codelists.io/codelist/196) currently supports accessibility metadata, but it's not frequently used yet. [MARC](https://www.loc.gov/marc/mac/2017/2017-11.html) is drafting up new fields to support accessibility metadata. When these fields become regularly used, it'll be useful to parse this data into Drupal.
